From 057ff2624aae585eebd6cefef8404d558da4c302 Mon Sep 17 00:00:00 2001
From: Ferhat Sobay <ferhat.sobay@yahoo.com.tr>
Date: Tue, 25 Sep 2012 16:07:19 +0300
Subject: [PATCH] Update guide/appendix/clients.textile

---
 guide/appendix/clients.textile |    1 +
 1 file changed, 1 insertion(+)

diff --git a/guide/appendix/clients.textile b/guide/appendix/clients.textile
index 44385d3..3e5dad1 100644
--- a/guide/appendix/clients.textile
+++ b/guide/appendix/clients.textile
@@ -24,6 +24,7 @@ h2. Clients
 * "ESClient":https://github.com/eriky/ESClient: A lightweight and easy to use Python client for ElasticSearch
 * "scalastic":https://github.com/bsadeh/scalastic: Scala client
 * "rawes":https://github.com/humangeo/rawes: Python low level client
+* "Jest":https://github.com/searchbox-io/Jest: Java Rest client.
 
 
 h2. Integrations
-- 
1.7.10

